roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -651,6 +661,8 @@ impl TweezerDevice {
             });
         }
         self.default_layout = Some(layout.to_string());
+        self.switch_layout(layout)
+            .expect("Internal error: switching to default layout failed");
```

Comment:
Should this never be allowed to fail? The function already returns a result, wouldn't it be safer to use `?` here?